### PR TITLE
Bsp orchestrator kustomization

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1352

### Change description ###

Bsp orchestrator kustomization

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/bsp/bulk-scan-orchestrator/aat.yaml [hunk 1]
```diff
@@ -3,32 +3,17 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: bulk-scan-orchestrator
-  namespace: bsp
   annotations:
-    flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:prod-*
     fluxcd.io/automated: "true"
     repository.fluxcd.io/java: java.image
```
Comment (adusumillipraveen) [inline k8s/namespaces/bsp/bulk-scan-orchestrator/aat.yaml:8]:
```suggestion
    
```

File: k8s/namespaces/bsp/bulk-scan-orchestrator/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-orchestrator
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
```
Comment (adusumillipraveen) [inline k8s/namespaces/bsp/bulk-scan-orchestrator/prod.yaml:8]:
```suggestion
```

File: k8s/namespaces/bsp/bulk-scan-orchestrator/prod.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    filter.fluxcd.io/java: glob:prod-*
```
Comment (adusumillipraveen) [inline k8s/namespaces/bsp/bulk-scan-orchestrator/prod.yaml:9]:
```suggestion
```

File: k8s/namespaces/bsp/bulk-scan-orchestrator/prod.yaml (hunk 2 above)
Comment (mdayican) [reply k8s/namespaces/bsp/bulk-scan-orchestrator/prod.yaml]: @aliveni checkout

**Pull request merged**